### PR TITLE
Added example of the order() function in the docs

### PR DESCRIPTION
**Pull request by stanislavkozlovski**
First PR here, please tell me if something is wrong

### CODE REVIEW

File: diesel/src/query_dsl/order_dsl.rs [hunk 1]
```diff
@@ -12,6 +12,28 @@ use query_source::QuerySource;
...
+/// #     }
+/// # }
+/// #
+/// # fn main() {
+/// #     use diesel::result::Error::NotFound;
+/// let connection = establish_connection();
+/// use self::users::dsl::*;
+/// // load the users, ordered by their ID descending and then by their name descending.
+/// // note: the id and name fields are imported from self::users::dsl::* and use the name of the columns in the DB, not in the struct.
+/// users.order((id.desc(), name.desc())).load::<User>(&connection).unwrap();
```
Comment (killercup) [inline diesel/src/query_dsl/order_dsl.rs]: `User` does not exits or is not in scope here. Maybe copy it from another example? (This is why CI is failing, btw.)

File: diesel/src/query_dsl/order_dsl.rs [hunk 2]
```diff
@@ -12,6 +12,30 @@ use query_source::QuerySource;
...
+/// # }
+/// #
+/// # fn main() {
+/// let connection = establish_connection();
+/// #     connection.execute("DELETE FROM users").unwrap();
+/// connection.execute("INSERT INTO users (name) VALUES ('Saul'), ('Steve'), ('Stan')").unwrap();
+/// use self::users::dsl::{users, id, name};
+/// // load all users' names, ordered by their name descending
+/// let ordered_names = users.select(name).order(name.desc()).load(&connection).unwrap();
+/// assert_eq!(Ok(vec![String::from("Steve"), String::from("Stan"), String::from("Saul")]), ordered_names);
```
Comment (killercup) [inline diesel/src/query_dsl/order_dsl.rs]: No need to wrap it in an `Ok`, you already called unwrap :)

Comment (killercup) [inline diesel/src/query_dsl/order_dsl.rs]: Do you want to add another test below that has two Steves (and also output the ids) to show how to use order with tuples?

Comment (Enether) [inline diesel/src/query_dsl/order_dsl.rs]: Sorry it's a bit hard to write code in comments with nowhere to test it, haha. Will do!

Comment (killercup) [inline diesel/src/query_dsl/order_dsl.rs]: No worries! We're grateful for each contribution :) If it's only minor things (like an `unwrap`), I can fix that before merging

**Comment by Enether** [comment]
```
17 | let ordered_names = users.select(name).order(name.desc()).load(&connection).unwrap();
   |                                                           ^^^^ cannot infer type for `U`
   |
   = note: type annotations or generic parameter binding required
```
What's the proper type I should put here? `ordered_names: Vec<String>` ?

How can I run these tests locally before committing, I feel stupid relying on the CI

**Comment by killercup** [comment]
> What's the proper type I should put here? ordered_names: Vec<String> ?

Looks good.

Have a look at <https://github.com/diesel-rs/diesel/blob/master/CONTRIBUTING.md#setting-up-diesel-locally>. To test doc changes, run `cargo test --doc` in the `diesel` directory.

**Comment by killercup** [comment]
Thanks again!

**Pull request merged**